torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -166,6 +166,7 @@ ScalarType numpy_dtype_to_aten(int dtype) {
     case NPY_HALF: return kHalf;
     case NPY_INT32: return kInt;
     case NPY_INT16: return kShort;
+    case NPY_INT8: return kChar;
```

Comment:
question: is `char` signed by default for all architectures?

Reply:
Disregard my question, we have modified the code to [use `int8_t` already](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/Dispatch.h#L59)